Need to add github.com to "Tunnel with firewall" allowlist (#13819)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-networks/deploy-tunnels/tunnel-with-firewall.md

| ----------- | -------- | --------- | ------- |
| `api.cloudflare.com`        | 443  | TCP (HTTPS) | Allows `cloudflared` to query if software updates are available. |
| `update.argotunnel.com`     | 443  | TCP (HTTPS) | Allows `cloudflared` to query if software updates are available. |

> | `github.com`     | 443  | TCP (HTTPS) | Allows `cloudflared` to download the latest software binary in update. |

Reviewer: ```suggestion
| `github.com`     | 443  | TCP (HTTPS) | Allows `cloudflared` to download the latest release and perform a software update. |
```
